index.js

```diff
@@ -3,6 +3,11 @@ import postcss from 'postcss';
 import perfectionist from 'perfectionist';
 import postcssSafeParser from 'postcss-safe-parser';
 
+const SUPPORTED_SCOPES = [
```

Comment:
It would be better as a `Set` there too.

> It's a `const` there.

My request has nothing to do with the `const`.